helios-services/src/main/java/com/spotify/helios/master/ZooKeeperMasterModel.java

```diff
@@ -653,6 +653,13 @@ private RollingUpdateOp rollingUpdateTimedoutError(final RollingUpdateOpFactory
               RollingUpdateError.IMAGE_PULL_FAILED,
               metadata);
     }
+    if (!taskStatus.getContainerError().equals("")) {
```

Comment:
It should never return null. I guess `isNullOrEmpty()` looks nice and protects against NPEs. Otherwise, you could just use `isEmpty()`.